src/ng-select/lib/ng-select.component.ts

```diff
@@ -713,17 +709,6 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
         this.itemsList.markItem(item);
     }
 
-    onWidgetFocus() {
-      this._focusFirstTag();
-    }
-
-    onTagBlur($event) {
-      if (!$event.relatedTarget ||
-          ($event.relatedTarget && !$event.relatedTarget.classList.contains('ng-value'))) {
-        this.focusedTag = null;
-      }
-    }
-
```

Comment:
@geebru any impact on functionality

Reply:
These got duplicated during the merge so this is just removing the duplicate code.